web/src/test/java/no/nav/modiapersonoversikt/service/digdir/DigDirServiceImplTest.kt

```diff
@@ -42,21 +42,20 @@ internal class DigDirServiceImplTest {
 
     @Language("json")
     private val jsonResponse = """
-      {     
-        "aktiv": true,
-        "epostadresse": "julenissen@nordpolen.no",
-        "epostadresseOppdatert": "2022-08-31T10:15:30+02:00",
-        "epostadresseVerifisert": "2022-08-31T20:00+02:00",
-        "kanVarsles": false,
-        "mobiltelefonnummer": "12345678",
-        "mobiltelefonnummerOppdatert": "2022-08-31T20:00+02:00",
-        "mobiltelefonnummerVerifisert": "2022-08-31T20:00+02:00",
-        "personident": "10108000398",
-        "reservert": false,
-        "sikkerDigitalPostkasse": null,
-        "spraak": "nb",
-        "spraakOppdatert": "2022-08-31T20:00+02:00"
-      }
+                {
+                  "personident": "25839399971",
```

Comment:
[FYI] FNR er fiktiv. Men liten obs-obs likevel. :) 

Typisk bruker vi Aremark Testfamilien (10108000398), eller hvor det lar seg gjøre bare en tallrekken (12345678910). Det har ingen vesentlig betydning, men er bare gjort for å sikre at vi ikke ved ett uhell inkluderer ett gyldig fødselsnummer i fremtiden.